crates/sncast/tests/e2e/account/add.rs

```diff
@@ -502,3 +503,29 @@ pub async fn test_private_key_as_int_in_file() {
         )
     );
 }
+
+#[tokio::test]
+pub async fn test_empty_config_add_profile() {
+    let tempdir = tempdir().expect("Unable to create a temporary directory");
+    File::create(tempdir.path().join(CONFIG_FILENAME)).unwrap();
+    let accounts_file = "accounts.json";
+
+    let args = vec![
+        "--url",
+        URL,
+        "--accounts-file",
+        accounts_file,
+        "account",
+        "add",
+        "--name",
+        "my_account_add",
+        "--address",
+        DEVNET_PREDEPLOYED_ACCOUNT_ADDRESS,
+        "--private-key",
+        "0x456",
+        "--add-profile",
+        "random",
+    ];
+
+    runner(&args).current_dir(tempdir.path()).assert().success();
```

Comment:
I think this line doesn't necessarily confirm that the command was executed successfully. Could we possibly use `assert_stdout_contains`, or even ensure that the config file includes the new profile?